scheme-service/src/main/java/uk/gov/justice/laa/fee/scheme/feecalculators/CalculateMediationFee.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.justice.laa.fee.scheme.feecalculators;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import uk.gov.justice.laa.fee.scheme.entity.FeeEntity;
+import uk.gov.justice.laa.fee.scheme.model.FeeCalculation;
+import uk.gov.justice.laa.fee.scheme.model.FeeCalculationRequest;
+import uk.gov.justice.laa.fee.scheme.model.FeeCalculationResponse;
+
+/**
+ * Calculate the mediation fee for a given fee entity and fee data.
+ * */
+public final class CalculateMediationFee {
```

Comment:
Suggest MedationFeeCalculator to be consist with class naming:
```suggestion
public final class CalculateMediationFee {
```